### ISSUE TITLE
Unable to exec in a container, 403 Forbidden error

### ISSUE THREAD

**Issue by jconallen** [kubernetes-client/java#196]
Trying out one of the examples I am unable to exec a simple shell command in a container.  I am using 1.0.0-beta2.    			

```
		      ApiClient client = Config.fromToken(
				"https://xxx.xxx.xxx.xxx:8001", 
				"<some incredibly long token string>",
				false);
                    Exec exec = new Exec(client);
		    boolean tty = System.console() != null;
		    final Process proc = exec.exec(
		    		"default", 
		    		"kubetoy-7447675dbf-72lcj", 
		    		new String[] {"sh", "-c", "echo foo"}, 
		    		true, 
		    		tty );


		    new Thread(
		            new Runnable() {
		              public void run() {
		                try {
		                  ByteStreams.copy(System.in, proc.getOutputStream());
		                } catch (IOException ex) {
		                  ex.printStackTrace();
		                }
		              }
		            })
		        .start();

		    new Thread(
		            new Runnable() {
		              public void run() {
		                try {
		                  ByteStreams.copy(proc.getInputStream(), System.out);
		                } catch (IOException ex) {
		                  ex.printStackTrace();
		                }
		              }
		            })
		        .start();

		    proc.waitFor();
		    try {
		      // Wait for buffers to flush.
		      Thread.sleep(2000);
		    } catch (InterruptedException ex) {
		      ex.printStackTrace();
		    }

		    proc.destroy();
```

with the exception being:

```
java.net.ProtocolException: Expected HTTP 101 response but was '403 Forbidden'
	at com.squareup.okhttp.ws.WebSocketCall.createWebSocket(WebSocketCall.java:123)
	at com.squareup.okhttp.ws.WebSocketCall.access$000(WebSocketCall.java:40)
	at com.squareup.okhttp.ws.WebSocketCall$1.onResponse(WebSocketCall.java:98)
	at com.squareup.okhttp.Call$AsyncCall.execute(Call.java:177)
	at com.squareup.okhttp.internal.NamedRunnable.run(NamedRunnable.java:33)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
`
```

**Comment by kondapally1989** [comment]
this may be similar to https://github.com/kubernetes-client/java/issues/180
BearerToken missing in header in /util/src/main/java/io/kubernetes/client/util/WebSockets.java

**Comment by jconallen** [comment]
Yes, when I turn on debugging I don't see the BearerToken being passed as well.

**Comment by brendandburns** [comment]
Yeah, that's my guess too, waiting on a fix from @ceeaspb but if it doesn't show up in a few days, I may just kick one in myself...  In the meantime #180 describes the needed change if you want to recompile...

**Comment by brendandburns** [comment]
I just merged #202 can you check at HEAD? I will build new maven libs once there is confirmation of the fix.

**Comment by jconallen** [comment]
This fixed it for me thanx!
